tests/sklearn/test_sklearn_autolog.py

```diff
@@ -527,6 +531,119 @@ def test_meta_estimator_fit_performs_logging_only_once():
         assert len(mlflow.search_runs([run._info.experiment_id], query)) == 0
 
 
+@pytest.mark.parametrize(
+    "estimator_class_and_space",
+    [
+        (sklearn.model_selection.GridSearchCV, {"kernel": ("linear", "rbf"), "C": [1, 5, 10]}),
+        (sklearn.model_selection.RandomizedSearchCV, {"C": uniform(loc=0, scale=4)}),
+    ],
+)
+@pytest.mark.parametrize("backend", [None, "threading", "loky"])
+def test_parameter_search_estimators_produce_expected_outputs(estimator_class_and_space, backend):
+    mlflow.sklearn.autolog()
+
+    estimator_class, search_space = estimator_class_and_space
+    svc = sklearn.svm.SVC()
+    cv_model = estimator_class(svc, search_space, n_jobs=1, return_train_score=True)
```

Comment:
This should probably be set to `n_jobs` > 1, otherwise I'm not sure that the parallel backend is having an impact.

Reply:
Done!